packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,9 +1,22 @@
+## 45.15.1 - May 17, 2019
+
+### Fixed
+
+#### salesforcedx-vscode-apex
+
+- Fix endIndex for Trigger location ([PR #1346](https://github.com/forcedotcom/salesforcedx-vscode/pull/1346))
+
+#### salesforcedx-vscode-core
+
+- Fix for default dev hub username in org picker and test fix ([PR #1352](https://github.com/forcedotcom/salesforcedx-vscode/pull/1352))
```

Comment:
Something like `Fix org picker not listing scratch orgs related to a dev hub created without alias` would be more understandable.